src/decode.h

```diff
@@ -887,6 +891,15 @@ void CaptureStatsSetup(ThreadVars *tv, CaptureStats *s);
      ((p)->action |= a)); \
 } while (0)
 
+#define PKT_MAX_DECODED_LAYERS 16
+
+#define PACKET_INCREASE_CHECK_LAYERS(p) do {                                        \
```

Comment:
I think adding a macro to hides a return statement inside is not a great solution. I understand the desire to keep things tidy, but this is not the way.

Reply:
ok :-)